File: Base/Website/Allors/Client/Base/Data/InvokeResponse.ts
module Allors.Data {
    export class InvokeResponse implements ResponseError {
        hasErrors: boolean;
        errorMessage: string;
        versionErrors: string[];
        accessErrors: string[];
        missingErrors: string[];
        derivationErrors: Allors.Data.ResponseDerivationError[];
    }
}

File: Base/Workspace.Domain/Base/Workspace/Data/InvokeResponse.cs
namespace Allors.Workspace.Data {
    public interface InvokeResponse : ErrorResponse {
        bool hasErrors { get; set; }
        string errorMessage { get; set; }
        string versionErrors { get; set; }
        string accessErrors { get; set; }
        string missingErrors { get; set; }
        PullResponseDerivationError[] derivationErrors { get; set; }
    }
}